Modules/Core/Common/test/itkXMLFileOutputWindowTest.cxx

```diff
@@ -120,9 +99,39 @@ itkXMLFileOutputWindowTest(int argc, char * argv[])
     ++numLinesRead;
   }
 
-  ITK_TEST_EXPECT_EQUAL(numLinesRead, numLinesExpected);
+  std::cout << "File used: " << current_logger_filename << std::endl;
+  ITK_TEST_EXPECT_EQUAL(numLinesRead, required_lines_in_file);
+
+  std::cout << "Test finished: " << numLinesRead << " (read) == " << required_lines_in_file << "(required)"
+            << std::endl;
+  return numLinesRead == required_lines_in_file;
+}
+
 
+int
+itkXMLFileOutputWindowTest(int argc, char * argv[])
+{
+  if (argc != 2)
+  {
+    std::cerr << "Missing parameters." << std::endl;
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv) << " <filename>" << std::endl;
```

Comment:
According to the ITK SW guide, we should use square brackets for optional arguments. However, in this case, the filename is not optional, so please remove the `<>`.